src/W/Message.elm

```diff
@@ -183,7 +202,15 @@ view attrs_ children_ =
                    , HA.class "ew-font-text ew-text-base ew-font-medium"
                    , HA.class "ew-py-2 ew-px-4 ew-pr-6"
                    , HA.class "ew-bg-base-bg ew-rounded"
-                   , HA.class "ew-border-l-[6px] ew-border-0 ew-border-solid ew-border-current"
+                   , HA.class
+                        (if attrs.noBorder then
+                            ""
+
+                         else
+                            "ew-border-l-["
+                                ++ String.fromInt attrs.borderWidth
+                                ++ "px] ew-border-0 ew-border-solid ew-border-current"
+                        )
```

Comment:
Tailwind can't handle dynamic values like this (it generates classes at compile time).

So since we're using a dynamic value we should make it a `style`  tag like this:
```elm
HA.style "border-width" (String.fromInt attrs.borderWidth ++ "px")
```

Also - I _think_ we don't need the `ew-border-0`  here anymore.